cni/network/network_windows.go

```diff
@@ -257,6 +257,7 @@ func getPoliciesFromRuntimeCfg(nwCfg *cni.NetworkConfig) []policy.Policy {
 			InternalPort: uint16(mapping.ContainerPort),
 			VIP:          mapping.HostIp,
 			Protocol:     protocol,
+			Flags:        hnsv2.NatFlagsLocalRoutedVip, // iota'd, NatFlagsLocalRoutedVip = 1, uint32
```

Comment:
Is this a constant value ? What does this mean ? ( Just curious)

Reply:
It is a constant value set here https://github.com/microsoft/hcsshim/blob/298b31d151ab799c1d7686f8ee9eec1cb4213926/hcn/hcnpolicy.go#L82 . 
Was trying to match other code snippets in cni where they match the TCP/UDP to constant values as well.

Reply:
can you add a comment above this change saying "its to support hostport policy mapping"